.circleci/config.yml

```diff
@@ -9,23 +9,22 @@ jobs:
           keys:
             - node-cache-{{ checksum "package-lock.json" }}
```

Comment:
you need to update this to make the cache work again recommendation to also add a counter to evict the cache in case is needed,
`yarn-v1-{{ checksum "yarn.lock" }}`

Reply:
can just change to yarn.lock. main reason I had the prefix be `node-cache` to be package manager agnostic.

Reply:
yeah, name is just a suggestion, like I said I would still add the `v1`, It has happened before to me that we had to expire the cache because of reasons and increment that number is the easiest thing

Reply:
👍